tests/e2e-permissions.spec.ts

```diff
@@ -0,0 +1,375 @@
+    ).toBeDisabled();
+
+    await expect(
+      page.getByRole("button", { name: "Edit rule" }),
+    ).toBeDisabled();
+    await expect(
+      page.getByRole("button", { name: "Remove rule" }),
+    ).toBeDisabled();
+  });
+
+  test("Cannot interact with Storage", async ({ page }) => {
+    await gotoURL(page, "/ui/");
+    await page.getByText("Storage", { exact: true }).click();
+
+    // Storage Pools
+    await page.getByText("Pools", { exact: true }).click();
+    await expect(
+      page.getByRole("button", { name: "Create pool" }),
+    ).toBeDisabled();
+    await visitPool(page, poolName);
+    await expect(
+      page.getByRole("button", { name: "Delete pool" }),
+    ).toBeDisabled();
+    await page.getByTestId("tab-link-Configuration").click();
+    await expect(page.getByLabel("Description")).toBeDisabled();
+
+    // Storage Volumes
+    await page
+      .getByLabel("main navigation", { exact: true })
+      .getByRole("link", { name: "Volumes" })
+      .click();
+    await expect(
+      page.getByRole("button", { name: "Create volume" }),
+    ).toBeDisabled();
+
+    await visitVolume(page, volumeName);
+    await page.getByTestId("tab-link-Configuration").click();
+    await expect(page.getByLabel("Size", { exact: true })).toBeDisabled();
+
+    await page.getByTestId("tab-link-Snapshots").click();
+    await expect(
+      page.getByRole("button", { name: "Create snapshot" }),
+    ).toBeDisabled();
+    await expect(
+      page.getByRole("button", { name: "See configuration" }),
+    ).toBeDisabled();
+
+    // Custom ISOs
+    await page.getByText("Custom ISOs", { exact: true }).click();
+    await expect(
+      page.getByRole("button", { name: "Upload custom ISO" }),
+    ).toBeDisabled();
+
+    await page.getByPlaceholder("Search").click();
+    await page.getByPlaceholder("Search").fill(customISOName);
+    await expect(page.getByRole("link", { name: poolName })).toBeVisible();
+    await expect(
+      page.getByRole("button", {
+        name: "You do not have permission to create instances",
+      }),
+    ).toBeDisabled();
+    await expect(
+      page.getByRole("button", {
+        name: "You do not have permission to delete this volume",
+      }),
+    ).toBeDisabled();
+
+    //Buckets
+    await page.getByText("Buckets", { exact: true }).click();
+    await expect(
+      page.getByRole("button", { name: "Create bucket" }),
+    ).toBeDisabled();
+  });
+
+  test("Cannot interact with Images", async ({ page }) => {
+    await gotoURL(page, "/ui/");
+    await page.getByText("Images", { exact: true }).click();
+    await expect(
+      page.getByRole("button", { name: "Upload image" }),
+    ).toBeDisabled();
+  });
+
+  test("Cannot interact with the Project", async ({ page }) => {
+    await gotoURL(page, "/ui/");
+    await page.getByText("Configuration", { exact: true }).click();
+    await expect(page.getByLabel("Description")).toBeDisabled();
+  });
```

Comment:
Please add the "Create project" button is disabled in the navigations project dropdown.